src/script/auth/page/Index.test.tsx

```diff
@@ -57,6 +67,25 @@ class IndexPage {
 }
 
 describe('when visiting the index page', () => {
+  it('shows the title', async () => {
+    const indexTitle = `${t('authLandingPageTitleP1')} ${Config.getConfig().BRAND_NAME} - ${t(
+      'authLandingPageTitleP2',
+    )}`;
+    new IndexPage(
+      mockStoreFactory()({
+        ...initialRootState,
+        runtimeState: {
+          hasCookieSupport: true,
+          hasIndexedDbSupport: true,
+          isSupportedBrowser: true,
+        },
+      }),
+    );
+    await waitForExpect(() => {
+      expect(global.window.document.title).toBe(indexTitle);
```

Comment:
does this work correctly? i wouldnt necessarily expect this to get the correct title from the index page